test/azure/Expected/AcceptanceTests/ModelFlattening/modelflattening/models/__init__.py

```diff
@@ -0,0 +1,56 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+try:
+    from ._models_py3 import BaseProduct
```

Comment:
test/azure/Expected/AcceptanceTests/ModelFlattening/ no longer exists and should not be brought back by the PR